setup-utils/install-mxnet-fedora-python.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+######################################################################
+# This script installs MXNet for Python along with all required dependencies on a Fedora Machine.
+# Tested on Fedora 21.0 + distro.
+######################################################################
+set -e
+
+MXNET_HOME="$HOME/mxnet/"
+echo "MXNet root folder: $MXNET_HOME"
+
+echo "Installing basic development tools, atlas, opencv, pip, graphviz ..."
+sudo yum update
+sudo yum groupinstall -y "Development Tools" "Development Libraries"
+sudo yum install -y atlas atlas-devel opencv opencv-devel graphviz graphviz-devel
+
+echo "Building MXNet core. This can take few minutes..."
+cd "$MXNET_HOME"
+make -j$(nproc)
+
+echo "Installing Numpy..."
+sudo yum install numpy
+
+echo "Installing Python setuptools..."
+sudo yum install -y python-setuptools python-pip
+
+echo "Installing Python package for MXNet..."
+cd python; sudo python setup.py install
```

Comment:
no. I mean remove `cd python; sudo python setup.py install` and keep export PYTHONPATH...